plugin.php

```diff
@@ -198,13 +201,6 @@ function json_api_maybe_flush_rewrites() {
  * @param WP_JSON_Server $server Server object.
  */
 function json_api_default_filters( $server ) {
-
-	// Post meta.
-	$wp_json_post_meta = new WP_JSON_Meta_Posts();
-	add_filter( 'json_endpoints',    array( $wp_json_post_meta, 'register_routes'    ), 0 );
-	add_filter( 'json_prepare_post', array( $wp_json_post_meta, 'add_post_meta_data' ), 10, 3 );
-	add_filter( 'json_insert_post',  array( $wp_json_post_meta, 'insert_post_meta'   ), 10, 2 );
```

Comment:
Not sure how best to integrate the controllers. Maybe we don't?

We need to link from the post to meta in any case, which is probably best handled through having meta poke into the main post controller and add the data, but I can't think of a nice implementation there.